sfdc37/tests/test.bal

```diff
@@ -290,14 +297,16 @@ function testGetDeletedRecords() {
     log:printInfo("salesforceClient -> getDeletedRecords()");
 
     time:Time now = time:currentTime();
-    string endDateTime = time:format(now, "yyyy-MM-dd'T'HH:mm:ssZ");
+    string|error time1 = time:format(now, "yyyy-MM-dd'T'HH:mm:ssZ");
```

Comment:
Is there a chance of an error ever being returned here? Either way shouldn't we be failing the test if an error is returned instead of setting empty string later on?